bmlt-meeting-list.php

```diff
@@ -434,28 +482,27 @@ function get($url, $cookies = null) {
 
             return wp_remote_get($url, $args);
 		}
-
-		function get_all_meetings() {
```

Comment:
is this deliberately commented out?

Reply:
seems to be dead code,  but I wasn't sure.

Reply:
this is being referenced in partials/_bmlt_server_setup.php, bombing out with an error.